flang/unittests/Optimizer/Builder/Runtime/ReductionTest.cpp

```diff
@@ -89,27 +90,63 @@ TEST_F(RuntimeCallTest, genMaxValTest) {
 
 void testGenMinVal(
     fir::FirOpBuilder &builder, mlir::Type eleTy, llvm::StringRef fctName) {
+  FILE *file;
+  file = fopen("/home/psteinfeld/log", "a");
+  fprintf(file, "In testGenMinVal\n");
+  fclose(file);
   mlir::Location loc = builder.getUnknownLoc();
   mlir::Type seqTy =
       fir::SequenceType::get(fir::SequenceType::Shape(1, 10), eleTy);
   mlir::Type refSeqTy = fir::ReferenceType::get(seqTy);
   mlir::Value undef = builder.create<fir::UndefOp>(loc, refSeqTy);
   mlir::Value mask = builder.create<fir::UndefOp>(loc, seqTy);
-  mlir::Value min = fir::runtime::genMinval(builder, loc, undef, mask);
-  checkCallOp(min.getDefiningOp(), fctName, 3);
+  mlir::Value min = fir::runtime::genMinval(builder, loc, undef, false, undef,
+      mask);
+  checkCallOp(min.getDefiningOp(), fctName, 4);
 }
 
 TEST_F(RuntimeCallTest, genMinValTest) {
+  FILE *file;
+  file = fopen("/home/psteinfeld/log", "a");
+  fprintf(file, "In genMinValTest\n");
+  fclose(file);
```

Comment:
It looks like some debug code was left here (and the test is failing)

Reply:
@psteinfeld I was able to reproduce the failure locally on my Mac. I guess Mac's filesystem layout is different and also you are specifically referring to your home directory in linux(?).
After removing the debug log messages and the commented code, the test passes for me.

Reply:
Thanks, @kiranchandramohan.  You're the best.